### PR TITLE
[#1827] Extend finding resource in main searchbar

### LINKED ISSUE #1827: Add providers and resource organisation as a parameter by which a resource can be found
In the search box, please add resource provider and resource organisation as the parameter to search result by:

Test-Cyfronet offers more resources than 2

### PR DESCRIPTION
**Pull request by danielkryska**
- Services can be found by provider or resource organisation

On search `cyfronet` in top searchbar, all resources of `Test-Cyfronet #10` should appear (3 of them).

Closes #1827

### CODE REVIEW

**Review by kmarszalek — CHANGES_REQUESTED** [review]
Change Service -> Resource in commit msg and then in changelog entry
also:
- add some tests
- missing entry in the chaangelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Please view this file on the master branch, on stable branches it's out of date.
 ## [Unreleased]
 
 ### Added
+- Extend finding resource in main searchbar including providers (@danielkryska, goreck888)
```
Comment (kmarszalek) [inline CHANGELOG.md:12]: should only add to search or also highlight RO/RP on the returned list?

**Review by Marcelinna — APPROVED** [review]

File: app/models/service/search.rb [hunk 2]
```diff
@@ -7,8 +7,9 @@ module Service::Search
     # ELASTICSEARCH
     # scope :search_import working with should_indexe?
     # and define which services are indexed in elasticsearch
-    searchkick word_middle: [:name, :tagline, :description, :offer_names],
-      highlight: [:name, :tagline]
+    searchkick word_middle: [:name, :tagline, :description, :offer_names,
+                             :resource_organisation_name, :provider_names],
+      highlight: [:name, :tagline, :resource_organisation_name, :providers, :provider_names]
```
Comment (kmarszalek) [inline app/models/service/search.rb:12]: .

**Review by kmarszalek — APPROVED** [review]
i don't like providers and some aspects but fundamentally it is ok and it works. :|

**Review by michal-szostak — APPROVED** [review]

**Pull request merged**